products/ocp4/profiles/moderate-node.profile

```diff
@@ -39,8 +39,6 @@ description: |-
 # CM-6 CONFIGURATION SETTINGS
 # CM-6(1) CONFIGURATION SETTINGS | AUTOMATED CENTRAL MANAGEMENT / APPLICATION / VERIFICATION
 extends: cis-node
```

Comment:
This should be removed since it's already being extended by `moderate-node-rev-4`